package.json

```diff
@@ -81,6 +84,9 @@
   "babel": {
     "presets": [
       "react"
+    ],
+    "plugins": [
```

Comment:
The docs for **babel-plugin-transform-react-remove-prop-types** show the following config for `.babelrc`:
```json
{
  "env": {
    "production": {
      "plugins": ["transform-react-remove-prop-types"]
    }
  }
}
```

Without adhering to that structure, won't that plugin be applied regardless of the `ENV`?